test/runnable/gdb_baseclass_fields.d

```diff
@@ -0,0 +1,29 @@
+/*
+REQUIRED_ARGS: -g
+PERMUTE_ARGS:
+GDB_SCRIPT:
+---
+r
+set print pretty off
+echo RESULT=
+p *c
+---
+GDB_MATCH: RESULT=.*\{a = 1, b = 2\}
+*/
+
+class B {
+  uint a;
+}
+
+class C : B {
+  int b;
+}
+
+void main()
+{
+  C c = new C();
+  c.a = 1;
+  c.b = 2;
+  asm { int 3; }
```

Comment:
This is not nice to ldc and gdc, especially on other platforms. You can also just set a breakpoint in the gdb script.

Reply:
Done.